trade-finance-manager-api/src/v1/controllers/amendment.controller.js

```diff
@@ -27,6 +30,13 @@ const sendAmendmentEmail = async (amendmentId, facilityId) => {
           const automaticAmendmentVariables = { user, dealSnapshot, amendment, facilityId, amendmentId };
           // sends email and updates flag if sent
           await sendAutomaticAmendmentEmail(automaticAmendmentVariables);
+        } if (amendment?.ukefDecision?.managersDecisionEmail && !amendment?.ukefDecision?.managersDecisionEmailSent) {
```

Comment:
this `if` statement should start on the next line

Reply:
done